pkg/wrapper/bundlewrapper.go

```diff
@@ -21,6 +21,8 @@ import (
 
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/options"
```

Comment:
These need to be added to the Bazel BUILD file

Reply:
I thought gazelle was supposed to do it automatically? Either way, did it manually. Thanks